src/languages/lib/swift.js

```diff
@@ -0,0 +1,285 @@
+import { concat, either } from '../../lib/regex';
+
+// Keywords that require a leading dot.
+export const dotKeywords = [
+  /Protocol\b/, // contextual
+  /Type\b/, // contextual
+];
+
+// Keywords that may have a leading dot.
+export const optionalDotKeywords = [
+  /init\b/,
+  /self\b/
+];
+
+// Regular keywords and literals.
+export const keywords = [
+  /\bAny\b/,
+  /\bSelf\b/,
+  /\b_\b/, // pattern
```

Comment:
Too common, should be removed or will need a custom rule to be scored 0.  This is one nice thing about `keywords` it's very easy to alter their relevance individually.

Reply:
Done.